runTopMedFreeze8VariantCaller.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
```

Comment:
Is this here just for reference? I don't think it will show up in Dockstore (which is fine if so, just checking).

Reply:
Just for reference